### PR TITLE
Update team page

### PR DESCRIPTION
**Pull request by Peyton232**

### CODE REVIEW

**Review by andrewhooker2 — APPROVED** [review]

File: about/our-team.php [hunk 1]
```diff
@@ -33,180 +33,222 @@
...
                         </div>
                         <div class="cell-md-6">
                             <div class="inset-xl-right-70 inset-xl-left-70 inset-left-15 inset-right-15">
-                                <h6 class="h6-with-small"><a href="#"> Tyler Adam Martinez</a><span class="small text-silver-chalice">Treasurer</span></h6>
-                                <p>Tyler is a junior currently studying electrical engineering with a minor in biomedical engineering. He focuses on robotics and automation, and loves designing cool circuits and building electronics.</p>
+                                <h6 class="h6-with-small"><a href="#">Jacob Gomez</a><span class="small text-silver-chalice">Public Relations</span></h6>
+                                <p>Jacob is a current Computer Science student at UNT.Although he balances school and work he finds time for extracurriculars.He has always enjoyed being apart of different organizations and getting to know new people. He loves using all the different software to model, create and design things for the club or for his own personal projects.</p>
```
Comment (ntindle) [inline about/our-team.php:161]: Missing space after period

File: about/our-team.php [hunk 2]
```diff
@@ -220,45 +262,45 @@
                         </div>
                     </div>
                 </div>
+
                 <!-- Event Coordinator -->
```
Comment (ntindle) [inline about/our-team.php:223]: Swap this

Comment (Peyton232) [reply about/our-team.php:223]: swap what?

File: about/our-team.php [hunk 3]
```diff
@@ -220,45 +262,45 @@
...
                         </div>
                     </div>
                 </div>
-                <!-- Webmaster -->
+                <!-- Event Coordinator -->
+				<!--
                 <div class="cell-lg-12 bio-area">
                     <div class="range range-sm-middle">
-                        <div class="cell-md-3"><img src="/images/bio-pics/peyton-thibodeaux.jpg" alt="" width="360" height="404" class="img-responsive" />
+                        <div class="cell-md-3"><img src="/images/bio-pics/michelle-vargas.jpg" alt="" width="360" height="404" class="img-responsive" />
```
Comment (sebastian-king) [inline about/our-team.php:288]: What's happening down here? I think this is commented out but if its not needed we should just nuke it.

File: about/our-team.php [hunk 4]
```diff
@@ -33,184 +33,226 @@
...
+                </div>
+            </div>
+            <!-- Vice-president and Financial Director -->
+            <div class="cell-lg-12 bio-area">
+                <div class="range range-sm-middle">
+                    <div class="cell-md-3"><img src="/images/bio-pics/tyler-adam-martinez.jpg" alt="" width="360" height="404" class="img-responsive" />
+                    </div>
+                    <div class="cell-md-6">
+                        <div class="inset-xl-right-70 inset-xl-left-70 inset-left-15 inset-right-15">
+                            <h6 class="h6-with-small"><a href="#"> Tyler Adam Martinez</a><span class="small text-silver-chalice">Vice-president and Financial Director</span></h6>
```
Comment (sebastian-king) [inline about/our-team.php:91]: Can we change "Vice-president" to "Vice President" :)

File: about/our-team.php [hunk 5]
```diff
@@ -33,184 +33,226 @@
...
                         </div>
                         <div class="cell-md-6">
                             <div class="inset-xl-right-70 inset-xl-left-70 inset-left-15 inset-right-15">
-                                <h6 class="h6-with-small"><a href="#"> Nicholas Tindle</a><span class="small text-silver-chalice">Project Manager</span></h6>
-                                <p>Nicholas Tindle is a Computer Engineering student at UNT. He works in software engineering and loves hackathons. You can generally find him wearing a hat and probably a sweatshirt. He has a long history of collaboration with UNT Robotics as the first president, a loyal advisor, and now Project Manager. He has also served as an advisor to the Dean and is currently an officer of Engineering United. Nick has helped host numerous events at the university over the years. In his professional life, he works in data analysis, web development, and python scripting.</p>
+                                <h6 class="h6-with-small"><a href="#"> Abdus Samee</a><span class="small text-silver-chalice">Event coordinator</span></h6>
```
Comment (sebastian-king) [inline about/our-team.php:179]: Event coordinator -> Event Coordinator

**Comment by sebastian-king** [comment]
Can you compress the image files? Some of them are more than 1MB which will really slow down load times. I think I try to keep them all under 50kb for the bio images. FastStone Photo Resizer is a fantastic tool for it btw.

File: about/our-team.php (hunk 2 above)
Comment (sebastian-king) [reply about/our-team.php:223]: I think Nick is referring to the fact that Jesse is Aerospace Division Lead, not Event Coordinator :)

**Review by ntindle — APPROVED** [review]

**Review by sebastian-king — CHANGES_REQUESTED** [review]
Looks all good now except for the file sizes, so once that's fixed we are  good merge!

**Comment by Peyton232** [comment]
the big images have been compressed now

**Review by sebastian-king — APPROVED** [review]

**Pull request merged**